Tests/Export-Ini.Unit.Tests.ps1

```diff
@@ -63,7 +63,7 @@ Describe "Export-Ini" -Tag "Unit" {
             $defaultObject["Category2"]["Comment1"] = "Key1 = Value1"
             $defaultObject["Category2"]["Comment2"] = "Key2 = Value2"
 
-            $script:defaultFileContent = "[Category1]${lf}Key1 = Value1${lf};Key2 = Value2${lf}${lf}[Category2]${lf};Key1 = Value1${lf};Key2 = Value2${lf}"
+            $script:defaultFileContent = "[Category1]${lf}Key1 = Value1${lf};Key2 = Value2${lf}${lf}[Category2]${lf};Key1 = Value1${lf};Key2 = Value2${lf}${lf}"
```

Comment:
I ran the tests and they are broken because of this changes in lines:
* 84
* 97
* 105
* 129
* 138
* 156

Reply:
Things would be easier if I knew how to run them. The bot doesn't execute them anymore for this PR :(